Sources/CoreAardvarkTests/CoreAardvarkTests-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
```

Comment:
do we need this file? Can we delete? Or is it easier to keep this in case we need to add bridging imports in the future?

Reply:
I just let Xcode add it automatically, but I think you're right we can get rid of it until/unless we need it in the future.